src/screens/VisualTests/graphql.ts

```diff
@@ -53,7 +53,7 @@ export const FragmentLastBuildOnBranchBuildFields = graphql(/* GraphQL */ `
     }
     ... on CompletedBuild {
       result
-      testsForStatus: tests(statuses: $testStatuses) {
+      testsForStatus: tests(first: 1000, statuses: $testStatuses) {
```

Comment:
Does this limit the number of results? I would think we'd always want all test results for the build